src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+/**
+ * The Bhutu chatbot app
+ */
+
+import java.io.IOException;
+import duke.DukeException;
+import duke.Ui;
+import duke.Storage;
+import duke.Parser;
+import duke.Items;
+import duke.task.Task;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.task.Todo;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private static final String DATA_FOLDER = "./data";
+    private static final String DATA_FILE = "duke.txt";
+    
+    // All UI functionality
+    private final Ui ui;
+
+    // All saved data related functionality
+    private final Storage storage;
+
+    // All user input related functionality
+    private final Parser parser;
+
+    // the list of items
+    private Items items;
+
+    /**
+     * Duke Constructor
+     */
+    public Duke() throws DukeException {
+        ui = new Ui();
+        storage = new Storage(DATA_FOLDER, DATA_FILE);
+        parser = new Parser();
+        try {
+            items = new Items(storage.loadData());
+        } catch (DukeException e) {
+            Ui.printMessage("You have no previously saved items. Creating new list");
+            items = new Items();
+        }
+    }
+
+    /**
+     * entry point of the duke chatbot.
```

Comment:
Perhaps need an empty line between description and parameter section? 🤔 
Perhaps the first word need to start with uppercase? 🤔